client/container.go

```diff
@@ -350,7 +349,10 @@ func (c *container) handleMounts(ctx context.Context, request *tasks.CreateTaskR
 		}
 		for _, m := range mounts {
 			if spec.Linux != nil && spec.Linux.MountLabel != "" {
-				if ml := label.FormatMountLabel("", spec.Linux.MountLabel); ml != "" {
+				if spec.Linux.MountLabel != "" {
+					// NOTE: Same logic as label.FormatMountLabel("", spec.Linux.MountLabel)
```

Comment:
The line below (assigning `ml`) seems straightforward enough that this comment isn't necessary.